source/features/action-used-by-link.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+
+export const isActionPage = (): boolean => location.pathname.split('/', 3).join('/') === '/marketplace/actions';
+
+function init(): void {
+	const actionRepo = select('.d-block.mb-2')!
```

Comment:
The selector is too generic. Try selecting the icon in the sidebar and then getting its parent. Example:

```js
(select('......')!.parentElement as HTMLAnchorElement).pathname.slice(1);
```